tox.ini

```diff
@@ -7,16 +7,19 @@ testdeps =
 [tox]
 envlist =
     py{37,38,39,310}
-    py38-lint
-    py38-coverage
-    py38-docs
+    lint
+    coverage
+    docs
+
+ignore_basepython_conflict = true
```

Comment:
Is it really needed?

Reply:
Yeah, otherwise when you run it locally you get warnings such as:
```python
/home/zisis/.local/lib/python3.6/site-packages/tox/config/__init__.py:593: UserWarning: conflicting basepython version (set 3.8, should be 3.10) for env 'py310';resolve conflict or set ignore_basepython_conflict
  proposed_version, implied_version, testenv_config.envname
/home/zisis/.local/lib/python3.6/site-packages/tox/config/__init__.py:593: UserWarning: conflicting basepython version (set 3.8, should be 3.7) for env 'py37';resolve conflict or set ignore_basepython_conflict
  proposed_version, implied_version, testenv_config.envname
/home/zisis/.local/lib/python3.6/site-packages/tox/config/__init__.py:593: UserWarning: conflicting basepython version (set 3.8, should be 3.9) for env 'py39';resolve conflict or set ignore_basepython_conflict

```

Reply:
I can specify however basepython for each lint, docs, coverage instead of putting it in the parent testenv.

Reply:
Yeah it is more explicit like this.